### PR TITLE
Add HP Legacy products (DL180G6, DL360e Gen8) and fix some args

### PR DESCRIPTION
**Pull request by kimlulz**
- Added ProLiant DL180 Gen6 (590638-371), ProLiant DL360e Gen8 (661189-B21)
- Removed FlexLOM for DL380 Gen7 Because It has no FlexLOM until Gen7
- Renamed ProLiantD-L380-Gen7.yml → ProLiant-DL380-Gen7.yml

### CODE REVIEW

**Comment by kimlulz** [comment]
Renamed ProLiantD-L380-Gen7.yml → ProLiant-DL380-Gen7.yml

File: device-types/HPE/ProLiant-DL180-Gen6.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: HPE
+model: ProLiant DL180 Gen6
+slug: hp_dl180_g6
+u_height: 2.0
+is_full_depth: true
+part_number: 590638-371
+comments: '[HP ProLiant DL180 G6 QuickSpecs](https://www.hpe.com/psnow/doc/c04284378)'
+console-ports:
```
Comment (danner26) [inline device-types/HPE/ProLiant-DL180-Gen6.yaml:9]: Do you have any info on the `weight` and `airflow`? If so, please add that

File: device-types/HPE/ProLiant-DL360e-Gen8.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+manufacturer: HPE
+model: ProLiant DL360e Gen8
+slug: hp_dl360e_g8
+u_height: 1.0
+is_full_depth: true
+part_number: 661189-B21
+comments: '[HP ProLiant DL360e Gen8 QuickSpecs](https://www.hpe.com/psnow/doc/c04284501)'
+console-ports:
```
Comment (danner26) [inline device-types/HPE/ProLiant-DL360e-Gen8.yaml:9]: Do you have any info on the `weight` and `airflow`? If so, please add that

File: device-types/HPE/ProLiant-DL180-Gen6.yaml (hunk 1 above)
Comment (kimlulz) [reply device-types/HPE/ProLiant-DL180-Gen6.yaml:12]: Added a `weight` based on the maximum set(all hard drives, power supplies, and processors installed)
```
weight:
 - value: 16.80
   unit: kg
```

File: device-types/HPE/ProLiant-DL360e-Gen8.yaml (hunk 2 above)
Comment (kimlulz) [reply device-types/HPE/ProLiant-DL360e-Gen8.yaml:12]: Added a weight based on the maximum set(all hard drives, power supplies, and processors installed)
```
weight:
 - value: 17.92
   unit: kg
````

**Review by danner26 — APPROVED** [review]

**Comment by danner26** [comment]
Thank you for the diligence! We appreciate your contribution

**Pull request merged**